src/visitors/pages/Signup.js

```diff
@@ -151,12 +156,20 @@ export default class Main extends Component {
         this.setState({ [name]: e.target.value });
         break;
     }
+
+    if (name === 'year') {
+      this.setState({ hasGivenAge: e.target.value !== BirthYear.nullValue });
+    }
   };
 
   createVisitor = (e) => {
     e.preventDefault();
 
-    Visitors.create({
+    if (!this.state.hasGivenAge && !this.state.ageCheck) {
+      return this.setState({ errors: { ageCheck: 'You must be over 13 to register' } });
```

Comment:
At what point do we inspect the value of the birth year and check that it makes the registering user above 13?